pages/open-reading-frames/[uuid].js

```diff
@@ -0,0 +1,141 @@
+// node_modules
+import PropTypes from "prop-types";
+// components
+import Breadcrumbs from "../../components/breadcrumbs";
+import {
+  DataArea,
+  DataItemLabel,
+  DataItemValue,
+  DataPanel,
+} from "../../components/data-area";
+import DbxrefList from "../../components/dbxref-list";
+import { EditableItem } from "../../components/edit";
+import JsonDisplay from "../../components/json-display";
+import ObjectPageHeader from "../../components/object-page-header";
+import PagePreamble from "../../components/page-preamble";
+import SeparatedList from "../../components/separated-list";
+// lib
+import buildBreadcrumbs from "../../lib/breadcrumbs";
+import { errorObjectToProps } from "../../lib/errors";
+import FetchRequest from "../../lib/fetch-request";
+import { isJsonFormat } from "../../lib/query-utils";
+
+function EnsemblLink({ ensemblid }) {
+  return (
+    <a href={`http://www.ensembl.org/Homo_sapiens/Gene/Summary?g=${ensemblid}`}>
+      {ensemblid}
+    </a>
+  );
+}
```

Comment:
It’d be nice to move this function to its own file so you can share this component between this and the genes page, and we don’t duplicate the code. I know it‘s not _quite_ duplicate, but you just have to pass the `Homo_sapiens` taxa for this one.

Reply:
done